scripts/lang/ca/__init__.py

```diff
@@ -89,6 +93,10 @@
     "q": "term(parts[-1])",
     # {{etim-s|ca|XIV}}
     "etim-s": "'segle ' + parts[2]",
+    # {{-etimologia-|ca|es}}
+    "-etimologia-": "strong('Etimologia:')",
```

Comment:
Hm this is nt good. Either there is something either nothing. We do not need to have "Etimologia:" printed. It is already handled homogeneously for all locales. Also, it does not seem to be the good handling for that template?

Reply:
It is the good handling... at least on wiktionary. See https://ca.wiktionary.org/wiki/Plantilla:-etimologia- and [Dijous](https://ca.wiktionary.org/w/index.php?title=dijous&action=raw) // [Dijous](https://ca.wiktionary.org/wiki/dijous) for example. 

But, sure, we don't really need it here. So let's remove it entirely.

Reply:
Oh yes, I misread the documentation. But indeed they are not needed :)